iOS/Projects/Features/SignUp/Sources/Data/Repository/ImageFormRepository.swift

```diff
@@ -42,118 +38,34 @@ public final class ImageFormRepository: ImageFormRepositoryRepresentable {
-      body.append("Content-Type: \(mimeType)\(lineBreak)\(lineBreak)".data(using: .utf8)!)
-      body.append(imageData)
-      body.append("\(lineBreak)".data(using: .utf8)!)
-      body.append("\(boundaryPrefix)".data(using: .utf8)!)
+    .map { imageForms in
+      return imageForms
     }
-
-    body.append("--\(boundary)--\(lineBreak)".data(using: .utf8)!)
-
-    return body
+    .eraseToAnyPublisher()
   }
-
-//  func sendURLRequest(boundary: String, body: Data) {
-//    guard let url = URL(string: "https://api.wonho.site/api/v1/images") else {
-//      print("Error: Invalid URL")
-//      return
-//    }
-//    var request = URLRequest(url: url)
-//    request.addValue("multipart/form-data; boundary=\(boundary)", forHTTPHeaderField: "Content-Type")
-//    request.httpMethod = "POST"
-//    request.httpBody = body
-//
-//    let task = URLSession.shared.dataTask(with: request) { data, response, error in
-//      if let error {
-//        print("Error: \(error)")
-//        return
-//      }
-//
-//      if let httpResponse = response as? HTTPURLResponse {
-//        print("Response Status Code: \(httpResponse.statusCode)")
-//      }
-//
-//      if let responseData = data {
-//        if let responseString = String(data: responseData, encoding: .utf8) {
-//          print("Response Data: \(responseString)")
-//        } else {
-//          print("Response Data cannot be decoded to UTF-8 string")
-//        }
-//      } else {
-//        print("No response data received")
-//      }
-//    }
-//
-//    task.resume()
-//  }
 }
 
 // MARK: - ImageFormEndPoint
 
-enum ImageFormEndPoint: TNEndPoint {
-  case image(Data, String)
+struct ImageFormEndPoint: TNEndPoint {
+  let headers: TNHeaders
+  var multipart: MultipartFormData?
+  init(imageDataList: [Data]) {
+    let uuid = UUID()
 
-  var boundary: String {
-    switch self {
-    case let .image(_, providedBoundary):
-      Log.make().debug("EndPoint boundary: \(providedBoundary)")
-      return providedBoundary
-    }
-  }
-
-  var path: String {
-    switch self {
-    case .image:
-      return "/api/v1/images"
-    }
-  }
-
-  var method: TNMethod {
-    switch self {
-    case .image:
-      return .post
-    }
-  }
-
-  var query: Encodable? {
-    return nil
-  }
-
-  var body: Encodable? {
-    switch self {
-    case let .image(formData, _):
-      Log.make().debug("EndPoint formData: \(formData)")
-      return formData
-    }
-  }
-
-  var headers: TNHeaders {
-    TNHeaders(headers: [
-      TNHeader(key: "Content-Type", value: "multipart/form-data; boundary=\(boundary)"),
-      TNHeader(key: "Accept-Encoding", value: "gzip"),
-      TNHeader(key: "Accept", value: "*/*"),
-      TNHeader(key: "Connection", value: "keep-alive"),
+    headers = .init(headers: [
+      .init(key: "Content-Type", value: "multipart/form-data; boundary=\(uuid.uuidString)"),
     ])
```

Comment:
P3: 만들어진 함수를 사용하면 더 좋을 것 같아요.!
```suggestion
    headers = [.contentType("multipart/form-data; boundary=\(uuid.uuidString)")]
```